rtengine/improccoordinator.cc

```diff
@@ -1646,8 +1646,10 @@ void ImProcCoordinator::updatePreviewImage(int todo, bool panningRelatedChange)
 
                 // Computing the internal image for analysis, i.e. conversion from WCS->Output profile
                 delete workimg;
+                workimg = nullptr;
```

Comment:
Maybe it's possible to convert `workimg` to a `std::unique_ptr<>` to get rid of the `delete`?

Reply:
Certainly, but I decided against doing it here to keep the PR focussed on certain classes of lgtm.com alerts. There are a bunch of alerts concerning possibly missing deletions which would suggest using `unique_ptr` and friends which I plan to look into in a separate PR.